index.html

```diff
@@ -300,7 +300,7 @@ <h3 class="hello"><em>bye!</em></h3>
   <section class="featured">
     <div class="container">
       <a class="featured__link" href="https://github.com/iamdustan/smoothscroll">Polyfill on <strong>GitHub</strong></a>
-      <a class="featured__link" href="https://github.com/iamdustan/smoothscroll-polyfill">Polyfill on <strong>npm</strong></a>
+      <a class="featured__link" href="https://npmjs.org/smoothscroll-polyfill">Polyfill on <strong>npm</strong></a>
```

Comment:
Thanks for spotting this! Could you change it to `https://npmjs.org/package/smoothscroll-polyfill` (npm does a redirect)

Reply:
👍